main.go

```diff
@@ -90,10 +94,29 @@ func main() {
 	flag.Parse()
 	configureLogging()
 
-	log.Info().Msgf("starting unwrapper service on port: %d", *flagPort)
+	httpsMessage := ""
+	if *flagHttps {
+		httpsMessage = fmt.Sprintf(" and https port: %d", *flagHttpsPort)
+	}
+	log.Info().Msgf("starting unwrapper service on http port: %d%s", *flagHttpPort, httpsMessage)
 	loadUnwrappers()
 	http.HandleFunc("/", handler)
-	http.ListenAndServe(fmt.Sprintf(":%d", *flagPort), nil)
+
+	go func() {
```

Comment:
for users who just want the HTTP server (i.e. they don't set the `-tls` flag) the program will exit straight away because it's running in a goroutine

I say we invert it so that the HTTP server runs at the end (so it blocks) and we run the https server in a goroutine (if `-tls` flag is provided)

```suggestion

	if *flagHttps {
	    go func() {
		if *flagCertFile == "" || *flagKeyFile == "" {
			log.Fatal().Msgf("The -tls argument, requires -cert and -key to be supplied")
		}
		err := http.ListenAndServeTLS(fmt.Sprintf(":%d", *flagHttpsPort), *flagCertFile, *flagKeyFile, nil)
		if err != nil {
			log.Error().Err(err).Send()
		}
	    }()
	}
	
	err := http.ListenAndServe(fmt.Sprintf(":%d", *flagHttpPort), nil)
	if err != nil {
	    log.Error().Err(err).Send()
	}
```